src/Microsoft.Restier.Security/ApiPolicyActivator.cs

```diff
@@ -33,15 +33,6 @@ public Expression Expand(QueryExpressionContext context)
         {
             Ensure.NotNull(context, "context");
 
-            if (this.InnerHandler != null)
-            {
-                var result = this.InnerHandler.Expand(context);
-                if (result != null)
-                {
-                    return result;
-                }
-            }
-
             if (context.ModelReference == null)
             {
                 return null;
```

Comment:
Yeah, you are right. It MAY have inner expander from the provider to call. Thanks for the reminder.